layouts/partials/components/docs-content.html

```diff
@@ -610,6 +610,23 @@ <h4 id="docs-microsoft-hyper-v">Rackspace Server Virtualization by Microsoft Hyp
                             </div>
                         </div>
                     </div>
+                    <div class="card blue">
+                        <div class="card-content">
+                            <div class="card-title">
+                                <h4 id="docs-software-defined-datacenter">Rackspace Software Defined Datacenter</h4>
+                            </div>
+                            <div class="card-body">
+                                <div class="list">
+                                    <div class="list-column">
+                                        <ul>
+                                            <li><a href="/docs/sddc/sddc-business-handbook/">SDDC - Business Handbook</a></li>
+                                            <li><a href="/docs/sddc/sddc-enterprise-handbook/">SDDC - Enterprise Handbook</a></li>
+                                        </ul>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
```

Comment:
I believe Lane mentioned to me that this should be the first card on top of the first column. So I say lets move it up to top of its column.